src/components/Icons/styled.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import { Link } from "react-router-dom";
+
+export const IconLink = styled(Link)`
+  color: var(--red);
+  font-size: calc(1em + 8vmin);
+  filter: drop-shadow(1px 1px 1px #000);
+  fill-rule: #fff;
+  cursor: pointer;
+  transition: all 0.2s;
+  &:hover {
+    transform: scale(1.2);
+  }
+`;
+
+export const RulesLink = styled.div`
```

Comment:
This should be a `button` since it pops up a modal. You can also extend the IconLink styles and add an `as="button"`, so something like
```js
export const RulesButton = styled(IconLink).attrs({
  as: "button"
}`
  // css
`
```
could work.
https://stackoverflow.com/questions/55776961/styled-components-extend-styles-and-change-element-type